src/tracks/AutomationTrack.cpp

```diff
@@ -57,9 +57,11 @@ TrackView * AutomationTrack::createView( TrackContainerView* tcv )
 
 
 
-TrackContentObject * AutomationTrack::createTCO( const MidiTime & )
+TrackContentObject * AutomationTrack::createTCO(const MidiTime & _pos)
```

Comment:
FYI, `_pos` is old style. Please use `pos` in new code, or at least don't revert `pos` to `_pos`.

Reply:
Thanks, I'll fix it. I had seem some existing files with `_pos` and ended up thinking I was fixing the others, when it was the other way around :laughing: 

What was the underscore for in the old style?